packages/base/core/src/svg/icons/concept/nutanix_concept.svg

```diff
@@ -1,6 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 13.97">
-  <defs><style>.cls-1{fill:#113f6d;}</style></defs>
-  <path class="cls-1" d="M15.4.09A.36.36,0,0,1,15.63,0h4A.33.33,0,0,1,20,.33a.33.33,0,0,1-.13.26L13.93,5.76a.32.32,0,0,1-.23.09.35.35,0,0,1-.24-.1L11.44,4a.31.31,0,0,1-.09-.22.33.33,0,0,1,.12-.25Z"/>
-  <path class="cls-1" d="M15.4,13.88a.32.32,0,0,0,.23.09h4a.33.33,0,0,0,.2-.59L13.93,8.21a.33.33,0,0,0-.47,0l-2,1.76a.28.28,0,0,0-.09.22.33.33,0,0,0,.12.25Z"/>
-  <path class="cls-1" d="M4.6,13.89a.35.35,0,0,1-.22.08h-4A.33.33,0,0,1,0,13.64a.33.33,0,0,1,.1-.23L7.19,7.24A.3.3,0,0,0,7.31,7a.33.33,0,0,0-.1-.24L.12.58A.3.3,0,0,1,0,.33.33.33,0,0,1,.33,0H4.38A.31.31,0,0,1,4.6.09l7.64,6.65a.33.33,0,0,1,.11.25.32.32,0,0,1-.11.24Z"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 13.97"><defs><style>.cls-1{fill:#113f6d;}</style></defs><g><g><g id="Nutanix-113F6D"><path class="cls-1" d="M15.4.09A.36.36,0,0,1,15.63,0h4A.33.33,0,0,1,20,.33a.33.33,0,0,1-.13.26L13.93,5.76a.32.32,0,0,1-.23.09.35.35,0,0,1-.24-.1L11.44,4a.31.31,0,0,1-.09-.22.33.33,0,0,1,.12-.25Z"/><path class="cls-1" d="M15.4,13.88a.32.32,0,0,0,.23.09h4a.33.33,0,0,0,.2-.59L13.93,8.21a.33.33,0,0,0-.47,0l-2,1.76a.28.28,0,0,0-.09.22.33.33,0,0,0,.12.25Z"/><path class="cls-1" d="M4.6,13.89a.35.35,0,0,1-.22.08h-4A.33.33,0,0,1,0,13.64a.33.33,0,0,1,.1-.23L7.19,7.24A.3.3,0,0,0,7.31,7a.33.33,0,0,0-.1-.24L.12.58A.3.3,0,0,1,0,.33.33.33,0,0,1,.33,0H4.38A.31.31,0,0,1,4.6.09l7.64,6.65a.33.33,0,0,1,.11.25.32.32,0,0,1-.11.24Z"/></g></g></g></svg>
```

Comment:
**issue(blocking):** Nested `<g>` is useless. ID seems useless too. Could you please ask an optimize version?